ru/browser/browser/zenAppExtensionFields.ftl

```diff
@@ -1,4 +1,6 @@
+
 extension-firefox-compact-galaxy-name = Zen Galaxy
-extension-firefox-compact-galaxy-description = Тема с темной цветовой схемой и приятной галактикой касается!
-extension-firefox-compact-dream-name = Дзен Мечта
-extension-firefox-compact-dream-description = Тема в светлой цветовой гамме с приятными мечтательными оттенками!
+extension-firefox-compact-galaxy-description = Тема с темной цветовой схемой и приятным прикосновением галактики!
```

Comment:
Keep the color-scheme thing consistent with default Firefox themes' descriptions and make it more playful! 🛰️✨ 

```suggestion
extension-firefox-compact-galaxy-description = Тема в темной цветовой гамме с нотками межгалактической неизвданности!
```